components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -241,6 +242,67 @@ class GleanTest {
         }
     }
 
+    @Test
+    fun `test sending of metrics ping`() {
+        val server = MockWebServer()
+        server.enqueue(MockResponse().setBody("OK"))
+
+        EventsStorageEngine.clearAllStores()
```

Comment:
You don't need to clear the `EventsStorageEngine` but rather the `StringMetricStorageEngine` here :)

Reply:
True dat!  There was also already a test that checked that the `defaults` ping was sent:
`fun 'test sending of default pings'()`